tidy3d/components/data/monitor_data.py

```diff
@@ -1066,6 +1068,157 @@ def translated_copy(self, vector: Coordinate) -> ElectromagneticFieldData:
             **field_kwargs,
         )
 
+    def to_zbf(
+        self,
+        fname: str,
+        background_refractive_index: float,
+        n_x: int,
+        n_y: int,
+        units: UnitsZBF,
+        freq: Optional[float] = None,
+        mode_index: Optional[int] = None,
+        r_x: float = 0,
+        r_y: float = 0,
+        z_x: float = 0,
+        z_y: float = 0,
+        rec_efficiency: float = 0,
+        sys_efficiency: float = 0,
+    ) -> Tuple[ScalarFieldDataArray, ScalarFieldDataArray]:
+        """For a 2D monitor, export the fields to a Zemax Beam File (zbf).
+
+        The mode area is used to approximate the beam waist, which is only valid
+        if the beam profile approximates a Gaussian beam.
+
+        Parameters
+        ----------
+        fname : str
+            Full path to the .zbf file to be written.
+        background_refractive_index : float
+            Refractive index of the medium surrounding the monitor.
+        n_x : int
+            Number of field samples along x. Must be a power of 2, between 2^5 and 2^13 inclusive per Zemax's requirements.
+        n_y : int
+            Number of field samples along y. Must be a power of 2, between 2^5 and 2^13 inclusive per Zemax's requirements.
+        units : `UnitsZBF`
```

Comment:
double check the style guide. I think we use double backslash quotes inside of the docstrings or anything rendered by docs basically.  Also would be good to add to `.zbf` where it comes up.